frontend/src/features/room-session/hooks/useRoomConnection.ts

```
@@ -156,22 +156,27 @@ export function useRoomConnection({ roomName, displayName, joinKey }: UseRoomCon
+      RoomEvent.TrackUnpublished,
+      RoomEvent.TrackSubscribed,
+      RoomEvent.TrackUnsubscribed,
+      RoomEvent.LocalTrackPublished,
+      RoomEvent.LocalTrackUnpublished,
+      RoomEvent.TrackMuted,
+      RoomEvent.TrackUnmuted,
+      RoomEvent.TrackStreamStateChanged,
+      RoomEvent.TrackSubscriptionStatusChanged
+    ] as const;
+
+    refreshEvents.forEach((eventName) => room.on(eventName, refresh));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)